addons/web/static/src/views/graph/graph_model.js

```diff
@@ -501,9 +501,10 @@ export class GraphModel extends Model {
         const processedGroupBy = [];
         for (const gb of groupBy) {
             const { fieldName, interval } = gb;
-            const { sortable, type } = fields[fieldName];
+            const { sortable, type, store } = fields[fieldName];
             if (
-                !sortable ||
+                // many2many are groupable only when it is stored (cf. odoo/fields.py)
+                (!sortable && !(type === "many2many" && store)) ||
```

Comment:
what if a many2many is not store but is still sortable. Is it possible?

Reply:
It never happens https://github.com/odoo/odoo/commit/8c8f328e290c83c1e00d43636fb9b8620979f6cd